### ISSUE TITLE
Faraday Flat Params

### ISSUE THREAD

**Issue by bkubic** [andyjeffries/flexirest#33]
I can't fpr the life of me figure out how to set Faraday::FlatParamsEncoder in the Faraday config block or on a per session basis. I've tried:

```
Flexirest::Base.faraday_config do |faraday|
    faraday.adapter(:net_http)
    faraday.options.timeout       = 10
    faraday.headers['User-Agent'] = "Flexirest/#{Flexirest::VERSION}"
    faraday.options.params_encoder = Faraday::FlatParamsEncoder
end
```
and
```
Flexirest::Base.faraday_config do |faraday|
  faraday.adapter(:net_http)
  faraday.options.timeout       = 10
  faraday.headers['User-Agent'] = "Flexirest/#{Flexirest::VERSION}"
  faraday.params_encoder = Faraday::FlatParamsEncoder
end
```

but my get params in a hash still have the brackets in the request url, which is invalid with the API I'm using. Any help?

**Comment by bkubic** [comment]
Found the issue - created pull request #34

**Comment by andyjeffries** [comment]
I've merged #34 and added the RSpec tests in 1bc6d69738ffd515f8ab42fb2ad8edb2b59eaa93. Then it's been released as v1.3.10. Thanks for the issue and PR.
